src/Umbraco.Web.UI.Client/src/views/propertyeditors/blockeditor/blockcard/blockcard.component.js

```diff
@@ -0,0 +1,23 @@
+(function () {
+    "use strict";
+
+    angular
+        .module("umbraco")
+        .component("umbBlockCard", {
```

Comment:
Does this file/folder/component belong in the property editors folder? this is really just a component and there isn't any property editor called 'blockeditor'

Reply:
Also corrected now